style.css

```diff
@@ -0,0 +1,959 @@
+}
+
+.head h2 {
+  width: 315px;
+  height: 52px;
+  font-family: "Crete Round", serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 40px;
+  line-height: 52px;
+  align-items: center;
+  color: #091e42;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+#vector {
+  width: 700px;
+  height: 0;
+  border: 1px solid #212121;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  align-items: center;
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  padding-top: 114px;
+}
+
+#img {
+  background-image: url("./images/Img-Placeholder.png");
+  background-size: contain;
+  background-repeat: no-repeat;
+  width: 560px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  transition: transform 0.5s;
+}
+
+#img:hover {
+  transform: scale(1.1);
+}
+
+.post {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 0;
+  gap: 10px;
+  width: 308px;
+  height: 348px;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.post h2 {
+  width: 250px;
+  height: 48px;
+  font-family: "Crete Round", serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 30px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  color: #091e42;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.post p {
+  width: 448px;
+  height: 140px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 22px;
+  color: #091e42;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.post .server {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0;
+  gap: 8px;
+  width: 297px;
+  height: 40px;
+  flex: none;
+  order: 2;
+  flex-grow: 0;
+  transition: all 2s ease-out 0s;
+}
+
+.frame .server li {
+  list-style: none;
+  text-align: center;
+  padding: 10px 5px;
+  font-size: 15px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  color: #fff;
+  background: rgba(255, 255, 255, 0.24);
+  border: none;
+}
+
+.post .server li {
+  list-style: none;
+  text-align: center;
+  border: 1px solid rgb(172, 169, 169);
+  padding: 10px 5px;
+  font-size: 15px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  color: #091e42;
+}
+
+.post .server li:hover {
+  color: #fff;
+  background-color: #091e42;
+}
+
+.button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 10px 5px;
+  font-style: normal;
+  font-family: "Inter", sans-serif;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  width: 124px;
+  height: 48px;
+  border: none;
+  background: #f55800;
+  color: #eee;
+  flex: none;
+  order: 3;
+  flex-grow: 0;
+}
+
+.button:hover {
+  background: #fff5e1;
+  color: #f55800;
+}
+
+.button:active {
+  background: #e05304;
+  color: #f4f1f0;
+}
+
+.frame {
+  display: grid;
+  grid-template-columns: 369px 369px 369px;
+  justify-content: space-between;
+  padding-top: 50px;
+  gap: 24px;
+}
+
```

Comment:
Great job so far 👍,  kindly check the responsiveness of your frames, it is breaking. 👇
<img width="764" alt="image" src="https://user-images.githubusercontent.com/73854999/208180984-86984ee3-5d9e-4a7d-9773-7660d3ceea51.png">